packages/clay-core/src/typography/Heading.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React, {ElementType} from 'react';
+
+export type HeadingLevel = 1 | 2 | 3 | 4 | 5 | 6;
+
+export type WeightFont =
+	| 'lighter'
+	| 'light'
+	| 'normal'
+	| 'semi-bold'
```

Comment:
I used `semi-bold` to be maintain the CSS variable. See here: https://clayui.com/docs/css/utilities/markup-text.html#css-text-weights

Reply:
Yeah, I see, I think the semibold writing although grammatically correct doesn't seem to be defined like that at the code level eg CSS, in the fonts too. @pat270 what do you think?

Reply:
@matuzalemsteles I just want to make sure I'm understanding correctly. We want to change it to `semibold`?

Reply:
@pat270 yeah, I want your opinion on this if it makes sense.

Reply:
We could add an alias? This isn't the first time I encountered this question. I chose `semi-bold` because of 
 https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping, but I see it written both ways.

In the future if we add something like Extra Bold, would it be `font-weight-extra-bold` or `font-weight-extrabold`?